tests/config/bootstrap.php

```diff
@@ -48,14 +48,11 @@
     }
 }
 
-const TERMINUS_BIN_FILE = './terminus.phar';
```

Comment:
See above comment for TERMINUS_BIN_FILE changes in this file

Reply:
This looks different but it hasn't changed.  Just restored to a different location in the file.

Reply:
@uberhacker Is there a reason to move this part up in the file?

Reply:
It doesn't hurt?  I'm of the opinion that all constants should be defined at the top in case it might be needed in future code above where it's defined now.

Reply:
Constants at the top are good. If we want to reduce the scope of the variable to just the area where it is used, then it should be a variable, not a `const`.